starters/adapters/static/package.json

```diff
@@ -11,8 +11,7 @@
     ],
     "nextSteps": {
       "lines": [
-        " You must change the 'origin' url under staticAdapter",
-        " inside 'vite.config.ts'"
+        "You have to change the 'origin' url in ./adapters/static/vite.config.ts"
```

Comment:
for this kind of effect we need to lines
<img width="417" alt="Screenshot 2023-10-10 at 22 23 48" src="https://github.com/BuilderIO/qwik/assets/35845425/f3767777-e49a-4a21-a805-0258d4787b6c">

Reply:
ok i will revert that